python/lsst/daf/base/exampleLib.py

```diff
@@ -0,0 +1,12 @@
+from _exampleLib import *
```

Comment:
I certainly hope we can avoid that. It drastically limits the ability of the python linters to detect errors, as well as making the code harder to read. If typing the full namespace becomes onerous then one can use an abbreviation.